librz/bin/format/mach0/mach0_relocs.c

```diff
@@ -10,6 +10,37 @@
 
 #include "mach0_utils.inc"
 
+#define RELOCATION_INFO_SIZE 8 // sizeof(struct relocation_info)
+
+/**
+ * \param buf buffer of at least RELOCATION_INFO_SIZE bytes
+ */
+static void read_relocation_info(struct relocation_info *dst, ut8 *src, bool big_endian) {
+	dst->r_address = (st32)rz_read_at_ble32(src, 0, big_endian);
+	uint32_t field = rz_read_at_ble32(src, 4, big_endian);
+	if (big_endian) {
```

Comment:
QEMU has helper macroses like `EXTRACT` etc. We could think about something similar too.